plugins/synaptics-rmi/fu-synaptics-rmi-ps2-device.c

```diff
@@ -195,11 +195,9 @@ fu_synaptics_rmi_ps2_device_write_byte (FuSynapticsRmiPs2Device *self,
 			g_usleep (1000 * 10);
 		}
 		if (i >= 3) {
-			g_set_error_literal (error,
-					     FWUPD_ERROR,
-					     FWUPD_ERROR_NOT_SUPPORTED,
-					     "cannot write byte after retries");
-			return FALSE;
+			/* Just break without error return because FW may not return */
+			/* ACK for some command, ie. RESET command.*/
+			break;
```

Comment:
I'm not convinced just assuming success is a good idea here. Could we add a parameter to `fu_synaptics_rmi_ps2_device_write_byte()`, e.g. `gboolean allow_failure` which would be TRUE for `_RESET` and FALSE for the other commands?